guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -266,6 +266,14 @@ If the script fails due to missing or outdated packages, you must download and i
 # hash -d {foreman-maintain} service 2> /dev/null
 ----
 
+. If you have migrated content from Pulp 2 to Pulp 3, remove all Pulp 2 content.
++
+[options="nowrap" subs="attributes"]
+----
+# {foreman-maintain} content remove-pulp2
+----
+This removes Pulp 2 RPMs, content in `/var/lib/pulp/content/`, the mongo database, and migration content in the Pulp 3 database.
```

Comment:
Can we say uploaded and synchronized packages instead of rpm?

Also, we're not supposed to add 's' to pluralise words according to ibm style guide.

Reply:
Hi @maximiliankolb, thank you for the review!  I've applied the change that you've suggested. I've replaced "RPMs" with "uploaded and synchronized Pulp 2 packages". Please would this be okay? If yes, I'll apply the same change to the part talking about upgrading a connected Satellite. Thank you!

Reply:
Great, this works for me!

Reply:
Thank you! @maximiliankolb, @melcorr so do you think we can proceed to merging the branch? Thank you!